src/configurations/sources/amp/db-config.json

```diff
@@ -1,5 +1,8 @@
 {
   "name": "AMP",
   "displayName": "AMP",
-  "type": "amp"
+  "type": "amp",
+  "options": {
+    "tag": "client-sdk"
```

Comment:
what about make this specific to `sdkPlatformType: 'client' | 'server'` 

if `sdkPlatform` is undefined then it's not an `SDK` ?

Reply:
This way we don't over-engineer and try to see to far into the future and solve the exact problem at hand.

cc @saikumarrs @sandeepdsvs

Reply:
I would rather call it `sdkExecutionEnvironment` to be more appropriate.

Reply:
We should also consider starting an README.md that describes the responsibilities of these properties.

Reply:
I'm good with `sdkExecutionEnvironment` or `sdkEnvironment` or `sdkTarget` 👍

Reply:
Even better, we have this,
https://github.com/rudderlabs/rudder-integrations-config/blob/4a66fb51a43b21bd21ea1bc5e4c116322145be7b/src/schemas/sources/db-config-schema.json

I'm adding the new field there.